modules/react/breadcrumbs/stories/stories_VisualTesting.tsx

```diff
@@ -106,6 +106,61 @@ export const WithOverflowMenu = () => {
   );
 };
 
+export const WithOverflowMenuHavingTwoItems = () => {
+  interface Breadcrumb {
+    id: string;
+    link?: string;
+    text: string;
+  }
+
+  const [items] = React.useState<Breadcrumb[]>([
+    {id: '1', text: 'Home', link: '/'},
+    {id: '2', text: 'Second Link', link: '#'},
```

Comment:
```suggestion
    {id: '2', text: 'Second Link'},
```

Can we not having link in the last object? It should be the current item and do not generate caret after it.